src/Layouts/Layout.php

```diff
@@ -215,10 +215,14 @@ public function duplicate($key)
      */
     public function duplicateAndHydrate($key, array $attributes = [])
     {
+        $fields = $this->fields();
```

Comment:
Could this and the foreach be written more concisely as 

```php
$fields = clone $this->fields()->all();
```
and still fix the issue?

Reply:
That didn't work for me, because the internal field objects didn't get cloned but they get copied with reference.